drivers/led/Kconfig.pwm

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Seagate Technology LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config LED_PWM
+	bool "PWM LED driver"
```

Comment:
I think we should make this default `y` if any nodes with the compatible are enabled, for consistency with the desired behavior for all drivers, no?

```
default $(dt_compat_enabled,pwm-leds)
```

or so.

Reply:
Good idead. Added.

Reply:
> I think we should make this default y if any nodes with the compatible are enabled, for consistency with the desired behavior for all drivers, no?

I was wrong about this :(.

I thought we had agreed to do this for all drivers in https://github.com/zephyrproject-rtos/zephyr/issues/10621#issuecomment-618689638, but rereading the discussion I see now that I was wrong and there is no community consensus on this.

There are in fact votes against it from @pabigot and @nandojve  -- see #devicetree on slack for details.

Can I ask you to remove this and offer my apologies for wasting your time? Then I am +1.

Reply:
There is no need to apology @mbolivar. It is an honest mistake and I removed the default from Kconfig.